### PR TITLE
Implement CustomerProfilePage.jsx

### PR DESCRIPTION
**Pull request by anhmht**
#112 
 - Create CustomerProfilePage.jsx and get personal information of this user
screencapture-localhost-3000-customer-profile-2019-03-27-19_21_09

### CODE REVIEW

File: imports/ui/modules/customer/components/CustomerProfilePage.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+          </Col>
+          <Col span={6} />
+        </Row>
+      </div>
+    );
+  }
+}
+
+export default compose(
+  withTracker(() => {
```
Comment (hoadinh2010) [inline imports/ui/modules/customer/components/CustomerProfilePage.jsx]: Why we need to get the userId in withTracker

File: imports/ui/modules/customer/components/CustomerProfilePage.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  state = {}
+
+  render() {
+    const { userId } = this.props;
+
+    return (
+      <div className="customer-profile-page">
+        <h2>Thông tin cá nhân</h2>
+        <Row>
+          <Col span={6} />
```
Comment (hoadinh2010) [inline imports/ui/modules/customer/components/CustomerProfilePage.jsx]: Don't use the empty column, should use `offset` instead

File: imports/ui/modules/customer/components/CustomerProfilePage.jsx (hunk 1 above)
Comment (anhmht) [reply imports/ui/modules/customer/components/CustomerProfilePage.jsx]: At the first time I though it was the right place. I'll move it inside render() function

File: imports/ui/modules/customer/components/CustomerProfilePage.jsx (hunk 2 above)
Comment (anhmht) [reply imports/ui/modules/customer/components/CustomerProfilePage.jsx]: my bad, I used span and offset instead

**Pull request merged**